codespell_lib/data/dictionary.txt

```diff
@@ -14709,10 +14709,12 @@ thats;->that's
 thck->thick
 theather->theater
 theer->there
+theery->theory
 theese->these
 thefore->therefore
 theif->thief
 theifs->thieves
+theire->they're
```

Comment:
also `their`

Reply:
In the code I've seen it with this typo. Not used instead of their.
Anyway, I'm going to update it.